README.md

```diff
@@ -64,25 +74,31 @@ Following the download of the `jwql` repository, contributors can then install t
 conda update conda
 ```
 
-Next, activate the `base` environment:
+Next, activate the `base` or `root` environment (depending on your version of `conda`):
 
 ```
-source activate base
+source activate base/root
```

Comment:
Is this just here to make sure that users are starting in a clean environment? Should the command be `source deactivate` instead?

Reply:
@laurenmarietta If I recall correctly when I first made these instructions, this command doesn't necessarily have to be executed if the user already had `conda` installed, but if a user has just installed `conda` for the first time, a new environment can't be made until the `base`/`root` is activated(?)

Reply:
Oh, that would make sense then. Thanks for explaining!